unicodetools/data/linkification/dev/LinkFormattingTest.txt

```diff
@@ -0,0 +1,125 @@
+# LinkFormattingTest.txt
+# Date: 2025-12-16, 17:57:01 GMT 
+# © 2025 Unicode®, Inc.
+# Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+# For terms of use and license, see https://www.unicode.org/terms_of_use.html
+#
+# The usage and stability of these values is covered in https://www.unicode.org/reports/tr58/
+#
+# ================================================
+#
+# Format: Each line has the following fields, separated by semicolons.
+# Field 0: Scheme/host
+# Field 1: Path
+# Field 2: Query
+# Field 3: Fragment
+# Field 4: Result — with minimal escaping
+#
+# Empty lines, and lines starting with # are ignored.
+# Otherwise # is treated like any other character.
+# Spaces around the semicolons are ignored.
+#
+# The Path, Query, and Fragment will contain backslash escapes when characters would otherwise be 
+# internal syntax characters in *that* part. 
+# For example, consider ?a=b\&c&d=ef
+# This represents the key-value pairs:
+# - <"a", "b&c">
+# - <"d", "ef">
+# The \& in the value of the first key-value pair does not represent a separator between key-value pairs, 
+# but rather a literal & in the string value "b&c".
+# ================================================
+
+
+# Selected test cases
```

Comment:
This file has no test cases for formatting emails. Do we no longer need that because we no longer handle quoted email local parts where escaping is even possible?